src/rest-server/src/controllers/v2/token.js

```diff
@@ -58,7 +59,13 @@ const get = async (req, res, next) => {
 /**
  *  Get the token in AAD Mode.
  */
-const getAAD = async (req, res, next) => {
+const getAAD = async (err, req, res, next) => {
+  logger.info(JSON.stringify(err));
```

Comment:
`err` may be empty, should move to `if (err) {}` block
and better `logger.warn`